ab3d2_source/hireswall.s

```diff
@@ -228,11 +232,11 @@ screendividethru:
 				move.l	(a0)+,d3
 				swap	d3
 				add.l	#DivThreeTable_vb,a5
-				move.w	(a5),StripData
+				move.w	(a5),draw_StripData_w
```

Comment:
What denotes the difference between Draw_ and draw_ ?

Reply:
As per the proposed style here https://github.com/mheyer32/alienbreed3d2/issues/16 , it's the (assumed) visibility of the identifier. Anything beginning with Draw_ belongs to the drawing code but can be read/written or called from elsewhere. Anything beginning with draw_ belongs to the drawing code but is only used within the drawing code. This helps to indicate the "module" scope of the identifier, given that the identifiers may span several related files belonging in the same conceptual module but are not used outside that scope.

Reply:
FYI, Currently there are references in hires.s that clearly belong to other modules, the drawing code, that once all this identification has been carried out can be moved. This is why you'll sometimes see a "private" identifier being used there. I usually examine the context of each use of an identified during the rename before I decide if it's supposed to be public or private.